packages/help-center/src/components/help-center-contact-form.tsx

```diff
@@ -382,27 +383,33 @@ export const HelpCenterContactForm = () => {
 		}
 		switch ( mode ) {
 			case 'CHAT':
-				if ( showingSibylResults ) {
+				if (
+					showingSibylResults &&
+					( hasTranslation( 'Still chat with us' ) || locale === 'en' )
+				) {
 					return __( 'Still chat with us', __i18n_text_domain__ );
 				}
 			case 'EMAIL':
-				if ( showingSibylResults ) {
+				if ( showingSibylResults && ( hasTranslation( 'Still email us' ) || locale === 'en' ) ) {
```

Comment:
Ok, those strings are not yet translated, but will this show no string at all to not English users?